src/interfaces/particle_interface.jl

```diff
@@ -25,6 +25,7 @@ The second type of functions define a hard interface for `AbstractParticle`:
 These functions must be implemented in order to have the subtype of `AbstractParticle` work with the functionalities of `QEDprocesses.jl`.
 """
 abstract type AbstractParticle end
+Base.broadcastable(part::AbstractParticle) = Ref(part)
```

Comment:
Is the `=Ref(...)` the idiomatic way to do this? What does `broadcastable(::Ref)` look like and would it be better to use that?

Reply:
I think the proposed implementation is idiomatic: see https://docs.julialang.org/en/v1/manual/interfaces/#man-interfaces-broadcasting